actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -113,5 +113,10 @@ def all_application_helpers
         all_helpers_from_path(helpers_path)
       end
     end
+
+    # Provides a proxy to access helpers methods from outside the view.
+    def helpers
+      @_helper_proxy ||= view_context
```

Comment:
Just curious - why not use `view_context.some_helper_method` in the controller instead of `helpers.some_helper_method`?